assets/src/blocks/Accordion/AccordionFrontend.js

```diff
@@ -0,0 +1,183 @@
+      dataLayer.push({
+        'event': 'Close FAQ',
+        'Question': txt
+      });
+
+    } else {
+      // const panel = $('.accordion').next($('.panel'));
+      $('.panel').removeClass('visibility');
+      $('.accordion-headline:before').removeClass('rotate');
+      dataLayer.push({
+        'event': 'Expand FAQ',
+        'Question': txt
+      });
+
+    }
+
+  }
+}
+
+  handleReadMoreClick() {
+    window.onclick = e => {
+      let btnRead = $(e.target.parentNode).text().substring(0, 50) + '...';
+      dataLayer.push({
+        'event': 'Read More FAQ',
+        'Question': btnRead
+      });
+    }
+
+
+  }
+
+  componentDidMount() {
+    // Rendering component`s values depending on props
+    this._isMounted = true;
+    return () => {
+      isMounted = false;
+    }
+    this.handleCollapseClick();
+    this.onChangeContent();
+    console.log("component Did Mount OK");
+    console.log(this._isMounted);
+
+  }
+
+  componentWillUnmount() {
+    this._isMounted = false;
+    console.log("component Did UnMount OK");
+    console.log(this._isMounted);
+    this.handleCollapseClick();
+    this.handleReadMoreClick();
+
+
+  }
+
+  componentDidUpdate() {
+
+    console.log("componentDidUpdate OK");
+  }
+
+  onChangeContent() {
+    console.log("onChangeContent OK");
+
+    this.handleCollapseClick();
+    this.handleReadMoreClick();
+  }
+
+
+  render() {
+    const {
+      // className,
+      accordion_title,
+      accordion_description,
+      accordion_rows,
+      accordion_id,
+      accordion_headline,
+      accordion_text,
+      accordion_btn_text,
+      accordion_btn_url,
+      isEditing,
+    } = this.props;
+
+
+   {/* Change styles */}
+    // let style = this.props.style || 'minimal'; // Needed to convert existing blocks
+    // if (className) {
+    //   style = className.split('is-style-')[1];
+    // }
+
+    // let accordionClassName = `block accordion-block my-1 accordion-style-${style}`;
+    // if (isEditing) accordionClassName += ` editing`;
+
+    {/* Toggle panels accordion - bug event opens on second click because of the double onclick event */}
+      // let acc = document.getElementsByClassName('accordion');
+      // let p;
+      // for (p = 0; p < acc.length; p++) {
+      //   acc[p].addEventListener("click", function() {
+      //     this.classList.toggle("active");
+      //         let panel = this.nextElementSibling;
+      //           panel.style.display === "block" ? panel.style.display = "none" : panel.style.display = "block";
+
+      //           // panel.style.visibility === "hidden" &&  panel.style.opacity === "0"
+      //           // ? panel.style.visibility === "visible" &&  panel.style.opacity === "1"
+      //           // : panel.style.visibility === "hidden" &&  panel.style.opacity === "0";
+      //   });
+      // }
+
+
+    return (
+
+      <Fragment>
+        <section className="block accordion-block my-0 py-0">
+            <header>
```

Comment:
Same as in the editor, the `header` tag in other blocks is only around the title, and not the description, we probably want to keep this markup 🙂